SendLog.py

```diff
@@ -1668,6 +1669,43 @@ def run_scilab_func_request():
     return list_value
 
 
+# App route for getting scilab expression output for Expression Block
+
+@app.route('/getExpressionOutput', methods=['POST'])
+def run_scilab_func_expr_request():
+    (diagram, __) = get_diagram(get_request_id())
+    if diagram is None:
+        print('no diagram')
+        return
+
+    pathfortxtfile = diagram.sessiondir;
```

Comment:
```suggestion
    pathfortxtfile = diagram.sessiondir
```

do not add a semicolon here